Makefile

```diff
@@ -31,3 +32,7 @@ $(PLATFORM_INFO):
 protoc:
 	protoc -I. --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative agent/agent.proto
 	protoc -I. --go_out=./pkg --go_opt=paths=source_relative --go-grpc_out=./pkg --go-grpc_opt=paths=source_relative manager/manager.proto
+
+mocks:
```

Comment:
We do not need mockery installation to be automated via Makefile - please keep Makefile simple. Do installation by hand, document this in the documentation. Use Makefile to trigger the actions, not to install SW.